src/newSeed/stage4.html

```diff
@@ -9,12 +9,11 @@
         <div class="infoInput">
           <label for="ppt">Funding Tokens per Seed Token</label>
           <div class="pptContainer">
-            <numeric-input
-              output-as-string="true"
+            <input
```

Comment:
Why did you switch this to input?  I don't see any reason, and it has to be a bignumber in wei.